src/dev_usb_linux.c

```diff
@@ -118,6 +118,12 @@ int open_dev_usb(struct device *dev)
 		}
 	}
 
+    // Lets ignore all devices with less than 6 axis to avoid the daemon to grab mice
```

Comment:
Is this change really necessary? The user can explicitly select a usb id they want to use with spacenavd, and while I agree that it doesn't make sense to attempt to use any device which is not 6-dof, I'd rather not artificially restrict this. Is the other change in dev.c not sufficient to fix the issue?

Reply:
Hi. You were right. I removed the blacklisting and pushed the commit.